package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "ptushki-frontend",
+  "proxy": "http://localhost:3001",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
+    "eslint": "eslint src/**/*.{ts,tsx}",
+    "eslint-fix": "npm run eslint -- --fix"
+  },
+  "browserslist": [
+    ">0.2%",
+    "not dead",
+    "not ie <= 11",
+    "not op_mini all"
+  ],
+  "dependencies": {
+    "bootstrap": "^4.3.1",
+    "ramda": "^0.26.1",
+    "ramda-adjunct": "^2.16.1",
+    "react": "^16.8.5",
+    "react-dom": "^16.8.5",
+    "react-redux": "^6.0.1",
+    "react-scripts": "2.1.8",
+    "reactstrap": "^7.1.0",
+    "redux": "^4.0.1",
+    "redux-actions": "^2.6.5",
+    "redux-devtools-extension": "^2.13.8",
+    "redux-thunk": "^2.3.0",
+    "typesafe-actions": "^3.2.1"
+  },
+  "devDependencies": {
+    "@types/jest": "^24.0.11",
```

Comment:
seems as redundant, you haven't test or their settings at all. this can be omittted. but take note, when you will setup tests that `ts-jest` isnt working properly with `jest@24` and same related it typing that would be non right.

Reply:
there is a sample test - App.test.tsx. And it runs ok for now. 
I would prefer not to omit this, because we want our jest globals to be typed as well as anything else.

Reply:
Btw, you are right, some things have changed -  there is no need now in ts-jest, ever since babel has support for ts, it works fine with default CRA config.

Reply:
I meant not omit but just use accordingly `@types/jest@23.x.x`. So if it works, this is great!